core/themes/basis/css/component/fieldset.css

```diff
@@ -24,7 +24,6 @@ fieldset .fieldset-legend {
   top: 0;
   width: 100%;
   padding: 0.5em 0.78947em; /* LTR */
-  text-transform: uppercase;
```

Comment:
I think this will affect all fieldset legends, which may break existing sites. Let's restore this line.

Reply:
Done 👍🏼